help/user-guide/isajson-templates.md

```diff
@@ -2,28 +2,38 @@
 title: SEEK User Guide - Experiment Sample Templates
 layout: page
 ---
+# Experiment Sample Templates
```

Comment:
I agree with moving here the text related to Templates that was in isa-json-compliant-experiment